cmake/CAOption.cmake

```diff
@@ -71,11 +72,11 @@ macro(ca_option name type description default)
   # Setup the main option of this macro.
   if(${type} STREQUAL "BOOL")
     option(${name} ${description} ${default})
-    message(STATUS "oneAPI Construction Kit ${description}: ${${name}}")
+    ca_message(STATUS CA_VERBOSE_DETAIL "oneAPI Construction Kit ${description}: ${${name}}")
```

Comment:
`STATUS` already indicates the verbosity, and CMake has ways of automatically suppressing noise based on the requested verbosity level (`CMAKE_MESSAGE_LOG_LEVEL`). We could use that.

Reply:
I thought about that, but it felt that we should output all warnings by default, and STATUS could have two levels i.e. summary information and detail information. The only one I felt has real meaning for me is the LLVM version when i run it. However I am open-minded we could just use STATUS etc as you suggest.

Reply:
The ones that you mark as verbose here do have meaning for me, I check them almost every time I need to re-run CMake to make sure I didn't leave some extension off that I will be trying to test.

Reply:
CMAKE_MESSAGE_LOG_LEVEL would be across LLVM too, whereas I think they want to only suppress OCK messages

Reply:
CMake variables have directory scope. If set in a subdirectory, it should only affect that subdirectory without affecting the rest of LLVM.

Reply:
I did try that, but I'll try again.